### PR TITLE
Add privacy information to the Subscriber Agreement question

### PR DESCRIPTION
**Pull request by pde**
Currently the Subscriber Agreement agreement interaction in the client does not address privacy issues directly (though they are part of Let's Encrypt's Subscriber Agreement).  This PR attempts to make Certbot's privacy stance clear in the same screen.  Not sure yet if this is too wordy, but it may be the best way forward.

### CODE REVIEW

**Comment by pde** [comment]
Here are some "before" screenshots of the current interaction:

pre
pre2

**Comment by pde** [comment]
And here are some "after" screenshots from this branch:

post
post2

**Comment by schoen** [comment]
I think the biggest privacy "surprise" for users is not necessarily the information that the CA collects, but rather the way the cert contents are automatically made public through CT. (Maybe people who've gotten certs from other non-CT-using CAs in the past would be especially surprised here.) If there's a way that we could usefully convey that to Certbot users, I would definitely advocate for it.

I haven't seen a huge amount of complaining about this issue, but I feel like a handful of people have mentioned it now and then, or at least been super-secretive about their cert contents as though they didn't know that that information had already been irrevocably published to the whole world!

**Comment by schoen** [comment]
Is there a reason to mention that the requested domain names are sent, but not the subject public key?

**Comment by pde** [comment]
Yes will add the public key.  I wonder if this should be broken down by "information that's inherently required for your public cert" (names, keys, server IP) and "information that is any sense optional" (email address and user agent)

**Comment by SwartzCr** [comment]
Looks like tests are failing due to linting.
I'm happy with this added wording, though would also be fine with a mention of CT logging

**Comment by pde** [comment]
I m now leaning against making this change; since we've documented all of this in the Certbot privacy policy, this would just make that screen much wordier and more confusing.
